rules.js

```diff
@@ -36,15 +40,14 @@ class RelativeLinks extends Rule {
   setup() {
     this.on("dom:ready", event => {
       const anchors = event.document.getElementsByTagName('a');
-      if (anchors === null) {
-        return;
-      }
-      anchors.forEach(a => {
-        const href = a.getAttribute("href");
-        if (href && href.value.startsWith("https://camel.apache.org")) {
-          this.report(a, `For links within camel.apache.org use relative links, found: ${href.value}`);
+      if (anchors !== null) {
+        for (let i = 0; i < anchors.length; i++) {
+          const href = anchors[i].getAttribute("href");
+          if (href && href.startsWith("https://camel.apache.org")) {
```

Comment:
Missing `href.value` here

```suggestion
          if (href && href.value.startsWith("https://camel.apache.org")) {
```

Reply:
Thanks for your help. I will add the href.value and modify the code accordingly

Reply:
This was not changed